frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+
+const Footer = () => {
+  return (
+    <>
+      <footer>
+        <NavLink exact to="/">
+          Home
+        </NavLink>
+        <NavLink to="/Wines">Wines</NavLink>
+        <NavLink to="/AboutUs">About Us</NavLink>
+        <div>Copyright 2021 - The greatest, rocking What the Wine Team.</div>
+      </footer>
+    </>
```

Comment:
No need for React fragment.
Also, we can wrap the links with a `nav` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav.
And you can use a list to list the nav links.

Reply:
Should I change the `div` here to `p`?